src/LinkDotNet.Blog.Web/Authentication/OpenIdConnect/AuthHelper.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Extensions.Configuration;
+using System;
+
+namespace LinkDotNet.Blog.Web.Authentication.OpenIdConnect;
+
+public static class AuthHelper
+{
+    public static string GetAuthProvider(IConfiguration configuration)
```

Comment:
I would prefer if this is a normal class that lives inside the DI - container.
Until now I used the `AppConfiguration` and `AppConfiugrationFactory` for such global settings. While this is not ideal and is open to discussion, I would prefer if we could stick to it for the time being

Reply:
Moved to app configuration and AppConfiugrationFactory.